README.md

```diff
@@ -47,29 +47,28 @@ chmod 777 config/ data/
 To start the Sons of the Forest server, navigate to the cloned repository's directory and use Docker Compose:
 
 ```bash
-docker-compose pull
-docker-compose up -d
+docker compose up -d --build
```

Comment:
The fact that it is not being built is intentional, so that not everyone who wants to use the container image has to build it.